frontend/ContextMenu.js

```diff
@@ -58,16 +63,39 @@ class ContextMenu extends React.Component {
     this.props.hideContextMenu();
   }
 
+  getHeight = element => {
```

Comment:
The changes to this file seem unrelated to the description of the PR. And I think they're actually included in another PR. Mind if we keep this PR focused on just the copy-name issue?